src/Api/Vault/AuthorizationHandlers/Collections/BulkCollectionAuthorizationHandler.cs

```diff
@@ -96,17 +100,23 @@ public class BulkCollectionAuthorizationHandler : BulkAuthorizationHandler<BulkC
     private async Task CanCreateAsync(AuthorizationHandlerContext context, IAuthorizationRequirement requirement,
         CurrentContextOrganization? org)
     {
-        // If the limit collection management setting is disabled, allow any user to create collections
-        // Otherwise, Owners, Admins, and users with CreateNewCollections permission can always create collections
+        // Owners, Admins, and users with CreateNewCollections permission can always create collections
         if (org is
-        { LimitCollectionCreationDeletion: false } or
         { Type: OrganizationUserType.Owner or OrganizationUserType.Admin } or
         { Permissions.CreateNewCollections: true })
         {
             context.Succeed(requirement);
             return;
         }
 
+        // Check for non-null org here: the user must be apart of the organization for this setting to take affect
+        // If the limit collection management setting is disabled, allow any user to create collections
+        if (org is not null && await GetOrganizationAbilityAsync() is { LimitCollectionCreationDeletion: false })
```

Comment:
suggestion (non-blocking): Could we instead pass the `org` into `GetOrganizationAbilityAsync()` to be extra explicit and make these `if ()` branches slightly less complicated and reduce their repetition in the other operations.

We can have `GetOrganizationAbilityAsync()` check for a null `org` and return `null` so we can still use the `await GetOrganizationAbilityAsync() is {...}` branch.

I also imagine we could do the same in the other handlers.

Reply:
This should be done now!